### PR TITLE
Feature abc 479 add is joined to page header

### PR DESCRIPTION
**Pull request by lgilmet**
Added storybook controls and implemented isJoined to add slds-page-header_joined. Works on vertical too.

Removed slds classes from title slot.

### CODE REVIEW

File: src/modules/base/pageHeader/__examples__/base/base.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
         icon-name="standard:opportunity"
         info="Info"
         variant="base"
+        is-joined={isJoined}
```
Comment (nawel-aa) [inline src/modules/base/pageHeader/__examples__/base/base.html:7]: The base example is normally as uncustomized as possible, but you can add `is-joined` to one of the other examples (`objectHome`, for example), and change it in the stories too.

File: src/modules/base/pageHeader/pageHeader.html [hunk 2]
```diff
@@ -65,11 +65,16 @@
                             <div class="slds-page-header__name-title">
                                 <template if:true={isBaseVariant}>
                                     <span
-                                        class="slds-page-header__title slds-truncate"
+                                        class="
+                                            slds-page-header__title
+                                            slds-truncate
+                                        "
                                     >
                                         <template if:true={hasStringTitle}>
```
Comment (nawel-aa) [inline src/modules/base/pageHeader/pageHeader.html:73]: The `if:true` can be placed directly on the new spans, we don't need the `<template>` tags anymore.

**Review by nawel-aa — CHANGES_REQUESTED** [review]
The broken tests should be fixed, and new ones should be added for the new attribute.
And don't forget to keep Confluence up to date! 🙂

File: src/modules/base/pageHeader/__examples__/base/base.html (hunk 1 above)
Comment (lgilmet) [reply src/modules/base/pageHeader/__examples__/base/base.html:7]: I just forgot it there after trying to make storybook work.

**Review by nawel-aa — APPROVED** [review]

**Comment by lgilmet** [comment]
Doc
- [ ] https://avonnicreator.atlassian.net/wiki/spaces/ABC/pages/1766457432/Page+Header

**Pull request merged**